examples/webvr_multiview.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webvr - ball shooter</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+		<style>
+			body {
+				font-family: Monospace;
+				background-color: #101010;
+				color: #fff;
+				margin: 0px;
+				overflow: hidden;
+			}
+			a {
+				color: #48f;
+			}
+		</style>
+	</head>
+	<body>
+		<script type="module">
+
+			import * as THREE from '../build/three.module.js';
+			import { BoxLineGeometry } from './jsm/geometries/BoxLineGeometry.js';
+			import { WEBVR } from './jsm/vr/WebVR.js';
+			import Stats from './jsm/libs/stats.module.js';
+
+			var container;
+			var camera, scene, renderer;
+
+			var room;
+			var stats;
+
+			var count = 0;
+			var radius = 0.02;
+			var normal = new THREE.Vector3();
+			var relativeVelocity = new THREE.Vector3();
+
+			var clock = new THREE.Clock();
+
+			init();
+			animate();
+
+			function init() {
+
+				container = document.createElement( 'div' );
+				document.body.appendChild( container );
+
+				//
+
+				var canvas = document.createElement( 'canvas' );
+				var context = canvas.getContext( 'webgl2', { antialias: false } );
```

Comment:
I think adding comment here about why these two lines are necessary would be helpful to users.

Reply:
Yep, what about something like:
```js
// Currently OVR_multiview2 is supported just on WebGL2 non-multisampled contexts
// so we must create the context manually ensuring `antialias` is set to false
//
// There is an ongoing discussion on how to add multisampled multiview support
// on the Khronos Group's WebGL repo: https://github.com/KhronosGroup/WebGL/issues/2912
// as soon as that will get solved we will update the code so it will be
// transparent for the user and you could use this extension with or without multisampled
// contexts
var context = canvas.getContext( 'webgl2', { antialias: false } );
```

Reply:
Looking good!

Reply:
thanks